src/api/ai_help.rs

```diff
@@ -269,6 +269,10 @@ fn log_errors_and_record_response(
     user_id: i64,
     help_ids: HelpIds,
 ) -> Result<Option<mpsc::UnboundedSender<CreateChatCompletionStreamResponse>>, ApiError> {
+    if !history_enabled {
+        return Ok(None);
+    }
```

Comment:
@fiji-flo I have second thoughts about early returning if `!history_enabled`, because that this function not only records the history, but also logs errors (including the `stream ended without a finish_reason` one).

Reply:
That should happen somewhere else. As in the new code path in the meta data PR.

Reply:
I agree. Let's wait for https://github.com/mdn/rumba/pull/424 to be merged and move the logic over in this PR afterwards.